### ISSUE TITLE
Hey, I see a lot of work there

### ISSUE THREAD

**Issue by MathisZerbib** [BenjaminDev/courtvision#28]
I'm glad you've made all of that by yourself, your exemple for Padel seems to be really nice, I would give it a try for my personnal project if you don't mind, feel free to contact me ! 👍

**Comment by chricarp** [comment]
Hi @BenjaminDev,
I want to say that this project is extremely interesting, congrats!

But I'm having some issues understanding how exactly do a test of this, maybe I'm missing something? @MathisZerbib did you manage to run the project?

Thanks in advance!

**Comment by MathisZerbib** [comment]
Yes of course but I didn't adapt for the paddle, I ran this only for tennis video

**Comment by chricarp** [comment]
Hi @MathisZerbib, I thought this project managed only padel footage.

Do you mind sharing how did you manage to run the project? Even on Tennis footage it's ok, I'm a novice in the world of Computer Vision and I'm trying to understand at best how this project works in practice.

thanks!

**Comment by MathisZerbib** [comment]
You Can reach me by email on my github
